tests/test_span.py

```diff
@@ -147,7 +147,7 @@ def test_traceback_without_error():
     assert not s.error
     assert not s.get_tag(errors.ERROR_MSG)
     assert not s.get_tag(errors.ERROR_TYPE)
-    assert "tb = ''.join(traceback.format_stack(limit=limit))" in s.get_tag(errors.ERROR_STACK)
+    assert "File" in s.get_tag(errors.ERROR_STACK)
```

Comment:
because we're keeping the line where the `set_traceback()` is called, I would say:
```
assert 'in test_traceback_without_error' in s.get_tag(errors.ERROR_STACK)
```